### DISCUSSION TITLE
Configure fails

### DISCUSSION THREAD

**Discussion by b1tw0rker** [PowerDNS/pdns#13369] [Q&A]
- [x ] This is not a support question, I have read [about opensource](https://www.powerdns.com/opensource.html) and will send support questions to the IRC channel, [Github Discussions](https://github.com/PowerDNS/pdns/discussions/) or the mailing list.
- [x ] I have read and understood the ['out in the open' support policy](https://blog.powerdns.com/2016/01/18/open-source-support-out-in-the-open/)

 - Program: dnsdist
 - Issue type: Bug report

### Short description
Configure fails coz of lua. But lua is available.

### Environment

 - Operating system: Rocky Linux release 9.2 (Blue Onyx) - x64
 - Software version: 1.8.2
 - Software source: compiled yourself

### Steps to reproduce
1. Download Source from tarball: https://downloads.powerdns.com/releases/ 1.8.2
2. Installe dependencies e.g devel packages
3. ./configure
4. ./configure --with-lua=/usr/bin/lua (second try)

### Expected behaviour
configure the program withou errors.

### Actual behaviour
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
checking for LUA... no
configure: error: No Lua not found, but is mandatory

Second try (with: ./configure --with-lua=/usr/bin/lua)

checking which Lua implementation to use... /usr/bin/lua
checking for LUA... no
configure: error: Selected Lua (/usr/bin/lua) not found

### Other information
In the dnsdist documentation is also a small but ichy bug.
Page: https://dnsdist.org/guides/dns-over-https.html

Missing:  ) - closet bracket

```
addDOHLocal('2001:db8:1:f00::1', '/etc/ssl/certs/example.com.pem', '/etc/ssl/private/example.com.key', "/dns", {customResponseHeaders={["x-foo"]="bar"}}
```

**Comment by Habbie** [comment]
Can you show what (Lua) packages you installed?

**Comment by Habbie** [comment]
> ./configure --with-lua=/usr/bin/lua (second try)

This is the wrong argument. Moving this to Discussions as it looks like a support question.

**Comment by Habbie** [comment]
> Missing: ) - closet bracket

Thanks! Want to send a Pull Request for that? The source is here: https://github.com/PowerDNS/pdns/blob/master/pdns/dnsdistdist/docs/guides/dns-over-https.rst#L28

**Reply by b1tw0rker** [reply]
I sended a Pull Request.
